src/js/hooks/inboundV2/addItems/useInboundAddItemsForm.jsx

```diff
@@ -576,6 +606,60 @@ const useInboundAddItemsForm = ({
     }
   };
 
+  const importTemplate = async (event) => {
+    spinner.show();
+    try {
+      const formData = new FormData();
+      const file = event.target.files[0];
+      const { stockMovementId } = getValues().values;
+
+      formData.append('importFile', file.slice(0, file.size, 'text/csv'));
+      const config = {
+        headers: {
+          'content-type': 'multipart/form-data',
+        },
+      };
+
+      await stockMovementApi.importCsv(stockMovementId, formData, config);
+
+      fetchLineItems(true);
+      if (_.isNil(_.last(getValues().values.lineItems)?.product)) {
```

Comment:
let's move some of the logic to separate variables as this `if` is a bit difficult to read